modules/dag-l1/src/main/scala/org/tessellation/dag/l1/infrastructure/trust/storage/TrustDbStorage.scala

```diff
@@ -0,0 +1,123 @@
+package org.tessellation.dag.l1.infrastructure.trust.storage
+
+import cats.effect.MonadCancelThrow
+import cats.syntax.functor._
+import cats.syntax.traverse._
+
+import org.tessellation.dag.l1.domain.trust.storage.TrustStorage
+import org.tessellation.dag.l1.infrastructure.db.Database
+import org.tessellation.schema.peer.PeerId
+import org.tessellation.schema.trust._
+
+import doobie.implicits._
+import doobie.quill.DoobieContext
+import io.getquill.context.Context
+import io.getquill.{Literal, SqliteDialect}
+
+object TrustDbStorage {
+  case class StoredTrust(
+    peerId: PeerId,
+    score: Option[Score],
+    rating: Option[Rating],
+    observationAdjustment: Option[ObservationAdjustment]
+  )
+
+  type Ctx = DoobieContext.SQLite[Literal] with Context[SqliteDialect, Literal]
+
+  def make[F[_]: MonadCancelThrow: Database]: TrustStorage[F] =
+    make[F](new DoobieContext.SQLite[Literal](Literal))
+
+  def make[F[_]: MonadCancelThrow](ctx: Ctx)(implicit db: Database[F]): TrustStorage[F] =
+    new TrustStorage[F] {
+      val xa = db.xa
+
+      import ctx._
+
+      val getStoredTrust = quote {
```

Comment:
`getStoredTrusts` (should be plural because that query returns multiple elements)

Reply:
Changed.